README.md

```diff
@@ -78,11 +80,29 @@ docker-compose build --pull --no-cache
 
 # recreate updated services
 docker-compose up -d
+```
+
+### Upgrade Greenlight
+**Important:** especially with a version before 2020-05-17 create a database backup before, since the data is not persitent between container recreations.
```

Comment:
```suggestion
**Important:** especially with a version before 2020-05-17 create a database backup before, since the data is not persistent between container recreations.
```
just a minor typo

Reply:
got already fixed some minutes ago ;)